### DISCUSSION TITLE
Is there a specific reason the get utils method not documented?

### DISCUSSION THREAD

**Discussion by ryan2559375** [react-hook-form/react-hook-form#5930] [Q&A]
So I have a use case where I have to check if a field is valid or not from the `useFormState` hook. It is very complicated to grab the `fieldState` from `useController` for various reasons. Basically what I did was 

```
const { errors } = useFormState();
const fieldInvalid = Boolean(get(errors, name));
```

I attached it for code review and understandably I was scrutinized for the use of the undocumented `get` method. Here I am wondering why is it not part of the documentation? Is it intended for internal purpose only? (but it is exported to the global react-hook-form)

Thank you.

I ended up writing my own useFieldState hook so it was accepted.

**Comment by bluebill1049** [comment]
Please use utils function from lib like lodash instead.
